Stateless/project.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "2.5.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.6.0"
+  },
+  "packOptions": {
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "projectUrl": "https://github.com/nblumhardt/stateless",
+    "iconUrl": "https://storage.googleapis.com/google-code-archive/v2/code.google.com/stateless/logo.png",
+    "requireLicenseAcceptance": false
+  },
+  "authors": [ "Nicholas Blumhardt and Contributors" ],
+  "description": "Create state machines and lightweight state machine-based workflows directly in .NET code",
+  "language": "en-US",
+  "title": "Stateless",
+  "buildOptions": { "keyFile": "Stateless.snk" },
+  "frameworks": {
+    "netstandard1.0": {
+      "imports": "dnxcore50",
```

Comment:
If I'm not mistaken, this will make all of the dnxcore50 APIs available, which effectively promotes the 1.0 standard up to ~1.5, so chances are it won't run on the very early platforms.